<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="checkNewReports">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>check New Reports</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewRead">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ReadLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reports Avalilable:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewReportsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>New Reports:</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewNewReports">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateReport">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Report</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditColumns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>521</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeDataSourceButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Data Source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
